### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by EzraEn1** [Joezo/atom-jshint#91]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (C:\Users\ezrae_000\AppData\Local\atom\app-0.192.0\resources\app\src\atom.js:55:13)
AtomJsHint.updateGutter (C:\Users\ezrae_000\.atom\packages\atom-jshint\lib\atom-jshint.js:172:26)
```

**Comment by eleidan** [comment]
Got the same issue with the following spec:
**OS:** Ubuntu 14.04.2
**Atom:** 0.200

**Comment by Joezo** [comment]
closed as duplicate of #70
